integrationTests/chainSimulator/staking/delegation_test.go

```diff
@@ -750,6 +764,358 @@ func testChainSimulatorCreateNewDelegationContract(t *testing.T, cs chainSimulat
 	require.Equal(t, blsKeys[0], hex.EncodeToString(unStakedKeys[0]))
 }
 
+func TestChainSimulator_MaxDelegationCap(t *testing.T) {
+	if testing.Short() {
+		t.Skip("this is not a short test")
+	}
+
+	roundDurationInMillis := uint64(6000)
+	roundsPerEpoch := core.OptionalUint64{
+		HasValue: true,
+		Value:    30,
```

Comment:
You have a lot of operations in an epoch. Suggestion to have this increased to 50